src/components/CatalogSearch/CatalogSearch.tsx

```diff
@@ -56,52 +57,57 @@ export interface CatalogSearchProps {
   onSubmit: FormEventHandler<HTMLFormElement>;
 }
 
+export const CatalogSearchInputs: FunctionComponent<
```

Comment:
Feels like this warrants a separate component?

Reply:
Yeah it would be more consistent with our file structure to place this in a separate file, will follow up